cmd/printers/printers.go

```diff
@@ -6,15 +6,21 @@ package printers
 import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"sigs.k8s.io/cli-utils/cmd/printers/printer"
+	"sigs.k8s.io/cli-utils/cmd/printers/table"
 	"sigs.k8s.io/cli-utils/pkg/apply"
 )
 
 const (
 	EventsPrinter = "events"
+	TablePrinter  = "table"
```

Comment:
Is this feature controlled by a flag? If yes, where is the corresponding change in cmd?

Reply:
It was added in a previous PR. The apply cmd calls `GetPrinter` and passes in the value of the `--output` flag.